### ISSUE TITLE
Please add option to look in /Day Planner/ folder instead of forcing Daily Note usage?

### ISSUE THREAD

**Issue by theandrewwade** [ivan-lednev/obsidian-day-planner#242]
Hi. Thanks for the great plugin. I've been using it for ages. I've noticed the 0.7 change to forcing using the Daily Note breaks something in my workflow. Perhaps others have this as well?

I plan for tomorrow the day before. My Daily note that I create every day pulls in events via iCalBuddy. I therefore need to start the Daily Note at the start of the day. 

My Day Plan is also really a guide, not very important later, and not something I keep for ever. I clean them out every so often. My Daily Note though is a critical note of what I actually did on the day. I do not delete these.

It would be significantly useful if Day Planner could look at a note other than the Daily note (like it used to) so that I can keep the "plan" separate from the Daily note, and be able to plan out the day before without a whole bunch of copy pasting the next morning, and also not cluttering up my Daily Note with Day Planner information.

**Comment by serenasarah** [comment]
Hi, I too love this plugin. However I've also had a problem working with - and when I posted on reddit asking for some help they sent me here.

Here's my issue: 

When I create a Day Planner Timeline View I keep getting the following message

Failed to read daily note template

Then when I click the empty timeline (ie none of my tasks appear) it creates a second daily note of the same date. This second note shows all the tasks in the timeline. However I get left with two daily notes for each day.

Do you know what's causing this? Any advice? 
Cheers.

**Comment by FeralFlora** [comment]
@serenasarah You should create a separate issue for this (as it is a separate issue).

**Comment by l5956578** [comment]
Yeah I would love to see this old feature make a comeback! Having the recurring / routine / reusable planner and linking it to the [day planner timeline] was perfect for giving a visual cue and burndown for multiple routines I keep (e.g. Planner AM, Planner PM, Planner Premarket, etc.). 

A small nit for me is that I use Daily Notes and Calendar (specifically the dots indicating I journaled on that day) as part of a journaling workflow. With the system using Daily Notes it sadly clutters my view indicating I journaled when it's actually now only reporting days that have any task defined in it.

I saw the fork where the older functionality is still available. I'm tempted just due to my use case. **BUT**....

The reason I hold off is that I really like your new aesthetic, and appreciate how subtasks are handled on the timeline. I see the direction this has taken and it's smart. Only wish we could bump up this request for all us old-school pre-0.7.0 users. 🌳🦖😄

![good_stuff](https://github.com/ivan-lednev/obsidian-day-planner/assets/96683430/5d6336a5-1285-4434-a403-c83f9d47bdee)

**Comment by ivan-lednev** [comment]
Hello, @l5956578!

> Having the recurring / routine / reusable planner and linking it to the [day planner timeline] was perfect for giving a visual cue and burndown for multiple routines I keep (e.g. Planner AM, Planner PM, Planner Premarket, etc.).

So you want a button that pins the timeline view to an arbitrary file, so that you can use the same file as a plan for many days?

**Comment by l5956578** [comment]
> So you want a button that pins the timeline view to an arbitrary file, so that you can use the same file as a plan for many days?

Yes, exactly! Pinning the timeline view to a file, so that the pinned file (i.e. my routine) shows its contents in the timeline view, is exactly what I would like.

This way, I could un-pin it and re-pin it at any time.
Or un-pin it and pin it to another routine. 
The pinned file would remain my timeline view until I un-pinned it.

I read thread #253 am also ok adding a yaml header, or any identifier really, to all of my routine files so the pinning action can locate available routines.

Thanks for the response! 🌟🌟🌟🌟🌟

**Comment by crossways** [comment]
Thumps up for this Issue

I really love your plugin and it helps me to focus on tasks and do better scheduling.
A feature to seperate the Day Planer from the Daily Note would be much appreciated. Like already described in this issue, me as well only can create Daily Notes on the exact day, due to some dependencies elsewhere. This causes trouble as soon I create a task within Day Planer days ahead. But planing days ahead now became crucial for my productivity

Thanks for your great work

**Comment by voiddata** [comment]
> Hi, I too love this plugin. However I've also had a problem working with - and when I posted on reddit asking for some help they sent me here.
> 
> Here's my issue:
> 
> When I create a Day Planner Timeline View I keep getting the following message
> 
> Failed to read daily note template
> 
> Then when I click the empty timeline (ie none of my tasks appear) it creates a second daily note of the same date. This second note shows all the tasks in the timeline. However I get left with two daily notes for each day.
> 
> Do you know what's causing this? Any advice? Cheers.

this is an issue which I am facing too
any fixes for this?

**Comment by Yodo9000** [comment]
I guess this is the kind of issue where you might use multiple vaults as a work around. (Though that also comes with downsides such as inter-vault linking not having the same level of support as normal linking.)

Also, this plugin has a [fork](https://github.com/ebullient/obsidian-day-planner-og) that brings back the old Day Planner folder.
